docs/API.md

```diff
@@ -715,29 +773,22 @@ Example response:
 
 An object with the following properties:
 
-* `"offset"`: An integer, optional. (default: 0).
-* `"limit"`: An integer, optional. (default: 200).
-* `"frontend_params"`: As described below.
+* `"offset"`: An *unsigned integer*, optional. (default: 0). Position of the first returned element from the database returned list.  
+* `"limit"`: An *unsigned integer*, optional. (default: 200). Number of element returned from the *offset* element.
+* `"frontend_params"`: An object.
 
-The value of `"frontend_params"` is an object in turn, with the
-keys `"domain"` and `"nameservers"`. `"domain"` and `"nameservers"`
-will be used to look up all tests for the given domain, separated
-according to if they were started with a `"nameservers"` parameter or
-not.
-
->
-> TODO: Do we have an SQL injection opportunity here?
->
-> TODO: Describe the remaining keys in the example
->
-> TODO: Describe the purpose of `"offset"` and `"limit"`
->
-> TODO: Is the `"nameservers"` value a boolean in disguise?
->
-> TODO: The description of `"frontend_params"` is clearly not up to date. Can it
-> be described in a better way?
->
+The value of "frontend_params" is an object with the following properties:
 
+* `"client_id"`: A *client id*, optional.
+* `"domain"`: A *domain name*, required.
+* `"profile"`: A *profile name*, optional.
+* `"client_version"`: A *client version*, optional.
```

Comment:
`client_id`, `client_version` and `profile` are ignored by the implementation, but they are not marked as deprecated here. If they aren't marked as deprecated an issue should be created to make the implementation respect them. If you like I can create the issue.